flor/interface/skipblock/readblock.py

```diff
@@ -25,10 +27,12 @@ def step_into(block_name: str, probed=False):
     def end(*args, values=None):
         lbracket = ReadBlock.pda.pop()
         block = journal.as_tree()[lbracket.sk].blocks[lbracket.gk]
+        return_args = []
```

Comment:
let's not co-mingle Readblock modifications with logger modifications. These should be separate PRs.

Reply:
Sure. I'll make another PR.